kernal/editor1.a65

```diff
@@ -69,7 +69,7 @@ CLSR	LDA HIBASE      ;FILL HI BYTE PTR TABLE
 	TAY
 	LDA #0
 	TAX
-LPS1	STY LDTB1,X
+LPS1	STY [b]LDTB1,X
```

Comment:
Is this by intention? LDTB1 evaluates to $D9, so that should not be necessary (does not hurt either).